CorsixTH/CMakeLists.txt

```diff
@@ -249,6 +249,24 @@ else()
   message("Lua modules are not linked and will be loaded at runtime")
 endif()
 
+# Find libpng
+if(VCPKG_TARGET_TRIPLET)
+  find_package(PNG REQUIRED)
```

Comment:
If these conditions are the same maybe we don't need the VCPKG branch? Confirmed the usage notes for libpng, and zlib.